docs/data/hubble/analyst-guide/queries-for-horizon-like-data.mdx

```diff
@@ -0,0 +1,922 @@
+  and selling_offer_id = <offer id>
+  and buying_offer_id = <offer id>
+  and selling_asset_code = <asset code>
+  and selling_asset_issuer = <asset issuer>
+  and selling_liquidity_pool_id = <liquidity pool id>
+  and selling_account_address = <account address>
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at between <start> and <end>
+```
+
+## Transactions
+
+[horizon/transactions](/docs/data/horizon/api-reference/resources/transactions)
+
+Transactions are commands that modify the ledger state and consist of one or more operations.
+
+Learn more about [transactions](/docs/learn/glossary/#transaction).
+
+### List All Transactions
+
+```sql
+select
+  id
+  , max_fee
+  , operation_count
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar.history_transactions`
+where true
+  -- optional filters
+  --and account = <account address>
+  --and ledger_sequence = <ledger sequence>
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at between <start> and <end>
+```
+
+### Retrieve a Transaction
+
+```sql
+select
+  id
+  , max_fee
+  , operation_count
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar.history_transactions`
+where true
+  and id = <transaction id>
+  -- transaction_hash is also available
+  --and transaction_hash = <transaction hash>
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at between <start> and <end>
+```
+
+### Retrieve a Transaction’s Operations
+
+```sql
+select
+  transaction_id
+  , op_id
+  , max_fee
+  , operation_count
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar_dbt.enriched_history_operations`
+where true
+  and transaction_id = <transaction id>
+  -- transaction_hash is also available
+  --and transaction_hash = <transaction hash>
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at between <start> and <end>
+```
+
+## Trade Aggregations
+
+[horizon/trade_aggregations](/docs/data/horizon/api-reference/aggregations/trade-aggregations)
+
+A trade aggregation represents aggregated statistics on an asset pair (base and counter) for a specific time period. Trade aggregations are useful to developers of trading clients and provide historical trade data.
+
+Hubble does not have as granular trade aggregations as Horizon. Data is available for a given date where trade aggregation information can be provided at the daily, weekly, monthly, or yearly level.
+
+### List Trade Aggregations
+
+```sql
+select
+  day_agg
+  , asset_a_code
+  , asset_a_issuer
+  , trade_count_daily
+  , trade_count_weekly
+  , trade_count_monthly
+  , trade_count_yearly
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar_dbt.trade_agg`
```

Comment:
do we trust this data enough to publicize it yet

Reply:
Hmm probably not. Will remove for now